north/src/runtime/mount.rs

```diff
@@ -431,3 +437,32 @@ async fn wait_for_file_deleted(path: &Path, timeout: time::Duration) -> Result<(
         .and_then(|r| r.map_err(Error::JoinError))
         .and_then(|r| r)
 }
+
+fn is_version_supported(manifest: &Manifest) -> bool {
+    const V1: Version = Version(semver::Version {
+        major: 1,
+        minor: 0,
+        patch: 0,
+        pre: vec![],
+        build: vec![],
+    });
+    const SUPPORTED_MIN_VERSION: &Version = &V1;
+    const KNOWN_MAX_VERSION: &Version = &V1;
+    const DEFAULT_VERSION: &Version = &V1; // used if manifest has no 'manifest_version' entry
+    let manifest_version = manifest
+        .manifest_version
+        .as_ref()
+        .unwrap_or(DEFAULT_VERSION);
```

Comment:
Is that safe? In case the manifest does not have a version, the npk might be rather recent or rather old...
OTOH if we disallow npks without a manifest version we will force everyone to rebuild their NPKs...
what do you think?

Reply:
If we always add manifest version from now on (I'll add it to the pack command), we can assume that all NPKs that are missing it are 'old' NPKs. In that case, we can assume version 1.0.0 for them.
In other words, the 'the npk might be rather recent' case should never occur, right?